orbstation/modules/cargo/orders.dm

```diff
@@ -3,3 +3,48 @@
 	desc = "Contains two Ham Helmets, one for you, and one for your beloved, be they a person or a corgi."
 	cost = PAYCHECK_CREW * 2
 	contains = list(/obj/item/clothing/head/helmet/ham,/obj/item/clothing/head/helmet/ham)
+
+/datum/supply_pack/organic/flatbread
+	name = "Flatbread Crate"
+	desc = "Gluten-Free! A taste of home for Tizarian's everywhere. \
+			Best prices this side of the galaxy! All deliveries are guaranteed to be 99% anomaly-free."
+	cost = CARGO_CRATE_VALUE * 10 // Best prices this side of the galaxy.
+	contains = list(/obj/item/pizzabox/imperial_flatbread,
+					/obj/item/pizzabox/rustic_flatbread,
+					/obj/item/pizzabox/italic_flatbread,
+					/obj/item/pizzabox/rustic_flatbread,
+					/obj/item/pizzabox/italic_flatbread,
+				)
+	crate_name = "flatbread crate"
+
+/datum/supply_pack/organic/mothicpizza
+	name = "Mothic Pizza Crate"
+	desc = "You'll be drawn to these Pizzas as if they were a flame! \
```

Comment:
I think this description could be better... but I can't think of anything better myself, so I probably shouldn't be throwing stones.

Reply:
i think the lizard one is going to be fine but the moth one is still eluding me yeah